### ISSUE TITLE
CALL FOR NEW DATA

### ISSUE THREAD

**Issue by caesar0301** [awesomedata/apd2#1]
Please follow the instruction to join one of the public data communities, 

[HOW_TO_CONTRIBUTE](https://github.com/awesomedata/apd2/wiki/HOW_TO_CONTRIBUTE-%E5%A6%82%E4%BD%95%E8%B4%A1%E7%8C%AE)

**Comment by SidVal** [comment]
It's not very clear.

Please add some example (with complete items).

**Comment by caesar0301** [comment]
Hi @SidVal, a quick note: Fork `apd-core` -> Add new data file -> Give a merge request.

Here is an example: https://github.com/awesomedata/apd-core/pull/2/commits/58b3f8357e0f1688b7ea689d25a29b301ec31c2c

**Comment by SidVal** [comment]
You have not understood me.

I mean that the document: https://github.com/awesomedata/apd-core/wiki/HOW_TO_CONTRIBUTE-%E5%A6%82%E4%BD%95%E8%B4%A1%E7%8C%AE not It is very clear, because the example you provide does not explain how each point is completed and what impact it has on the final file that the bot creates.

You could edit the Wiki better to add more documentation and that way more people would collaborate.

**Comment by loganbyers** [comment]
From having just added a dataset to APD, I can say there seems to be only four variables in the YAML that currently transfer to the APD readme:

* `title` -- used as the visible text
* `homepage` -- used as the link/href
* `category` -- used to place the dataset under the appropriate heading
* `description` -- appended to the `title` in the visible text

The YAML -> Markdown conversion looks like:

```markdown
[{title}]({homepage}) - {description}
```
 with `description` truncated as needed.

**Comment by ishandutta2007** [comment]
Why are so many pull requests kept in pending state inspite of passing travis ci build tests?

**Comment by caesar0301** [comment]
@ishandutta2007  Dont worry. [Let the Bullets Fly](https://www.rottentomatoes.com/m/let_the_bullets_fly/) :)

**Comment by ishandutta2007** [comment]
???

**Comment by ishandutta2007** [comment]
I think its better if you add a short comment mentioning reason, 
doing so will help new contributors be on the same page.

**Comment by caesar0301** [comment]
I explained why I gave labels to some PRs in https://github.com/awesomedata/apd-core/pull/67#issuecomment-443777879. I really want this to be a pure high-quality data sharing repo. No advertisement! No spam! No reputation promotion! Maybe, sometimes actually, I gave a wrong judgement about the quality of a request. I want you some people, jump out, and point out that I gave a wrong decision. I will fix my label when I am convinced and give each request a responsible review. Thank you all contributors!

**Comment by ishandutta2007** [comment]
My point is, if it fails your criteria it should fail the test right a way. is it too hard to automate it ?
The amount of time that you might have spent manually reviewing might be more than what writing a quick test to handle that scenario.

**Comment by caesar0301** [comment]
@ishandutta2007 Yes I know. Am trying to automate validation as best as I can. But to my programming experience, some criteria are hard to implemented as code, say fake submission for ads.

**Comment by ishandutta2007** [comment]
Create a pending validation list, and share it, let the open souce community help you on those too, I will see if I can do any.
